WordPress/Classes/ViewRelated/Comments/CommentDetailViewController.swift

```diff
@@ -85,8 +84,9 @@ class CommentDetailViewController: UIViewController, NoResultsViewHost {
         iconAttachment.image = Style.ReplyIndicator.iconImage
 
         let attributedString = NSMutableAttributedString()
-        attributedString.append(.init(attachment: iconAttachment, attributes: Style.ReplyIndicator.textAttributes))
-        attributedString.append(.init(string: " " + .replyIndicatorLabelText, attributes: Style.ReplyIndicator.textAttributes))
+        attributedString.append(NSAttributedString(attachment: iconAttachment))
+        attributedString.append(.init(string: " " + .replyIndicatorLabelText))
+        attributedString.addAttributes(Style.ReplyIndicator.textAttributes, range: NSMakeRange(0, attributedString.length))
```

Comment:
The icon used here is the the arrow icon at the beginning of "You replied to this comment".

| Before | After |
| ------ | ----- |
| <img width="556" alt="reply-before" src="https://github.com/user-attachments/assets/3ddc210b-fc49-464b-9f46-50ed0316c8d0"> | <img width="556" alt="reply-after" src="https://github.com/user-attachments/assets/73c7f0a0-5627-4e3e-9c27-5b78f7d064e7"> |